src/App/Utilities/PasswordStrengthHelper.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Bit.App.Utilities
+{
+    public static class PasswordStrengthHelper
+    {
+        public static List<string> GetPasswordStrengthUserInput(string email)
+        {
+            var atPosition = email?.IndexOf('@');
+            if (atPosition is null || atPosition < 0)
+            {
+                return null;
+            }
+            var rx = new Regex("/[^A-Za-z0-9]/", RegexOptions.Compiled);
+            var data = rx.Split(email.Substring(0, atPosition.Value).Trim().ToLower());
+
+            return new List<string>(data); ;
+        }
```

Comment:
I'd move this directly into the `IPasswordGenerationService` and maybe add a method on the `PasswordStrengthViewModel` for easy access so the service doesn't need to be injected on the `IPasswordStrengthable` implementations.
Just in case you have double ";" on the return line (19)